home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/RoleDto.java

```diff
@@ -4,7 +4,8 @@
 
     ADMIN("admin"),
     USER("user"),
```

Comment:
delete here too

Reply:
It should be deleted in role matrix impl